ethportal-api/src/utils/bytes.rs

```diff
@@ -0,0 +1,66 @@
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum UtilsError {
```

Comment:
I think it will be more clear if we rename this enum to something like `HexError`, and change the enum variant `HexPrefixError` to `PrefixError`.

Adding a short docstring for the enum will be a nice addition too.

Reply:
Done 👍